content/administration/maintain/google_oauth.rst

```diff
@@ -0,0 +1,240 @@
+configuration automatically loads the token in Odoo, and a graphic stating
+:guilabel:`Gmail Token Valid` appears in green.
+
+.. image:: google_oauth/green-token.png
+   :class: img-fluid
+   :align: center
+   :alt: Configure Outgoing Email Servers in Odoo.
+
+Finally, save the settings and :guilabel:`Test the Connection`.
+
+The Odoo database can now send safe, secure emails through Google using OAuth authentication.
+
+Google OAuth FAQ
+================
+
+Production VS Testing Publishing Status
+---------------------------------------
+
+Choosing :guilabel:`Production` as the :guilabel:`Publishing Status` (instead of Testing) will
+display the following warning message:
+
+.. image:: google_oauth/published-status.png
+   :class: img-fluid
+   :align: center
+   :alt: OAuth is Limited to 100 Sensitive Scope Logins.
+
+To correct this warning, navigate to the 
+`Google APIs platform <https://console.cloud.google.com/apis/credentials/consent>`__. If the app is
+in :guilabel:`Production`, then click :guilabel:`Back to Testing` to correct the issue.
+
+.. image:: google_oauth/back-to-testing.png
+   :class: img-fluid
+   :align: center
+   :alt: Back to Testing in Google OAuth.
+
+No Test Users Added
+-------------------
+
+If no test users are added to the OAuth consent screen, then a 403 access denied error will
+populate.
+
+.. image:: google_oauth/403-error.png
+   :class: img-fluid
+   :align: center
+   :alt: 403 Access Denied Error.
+
+To correct this error, return to the :guilabel:`OAuth consent screen` under
+:guilabel:`APIs & Services` and add test user(s) to the app. The email added should be the one
+being configured to send in Odoo.
+
+Gmail Module not Updated
+------------------------
+
+Should the :guilabel:`Gmail Module` in Odoo not be updated to the latest version, then a Forbidden
+Error message (You don't have the permission to access the requested resource. It is either
+read-protected or not readable by the server.) will populate.
+
+.. image:: google_oauth/forbidden-error.png
+   :class: img-fluid
+   :align: center
+   :alt: Forbidden Error in Google OAuth.
+
+To correct this error, go to the :guilabel:`Apps` module and clear the search terms out. Search for
+*Gmail* or *Google* and upgrade the :guilabel:`Google Gmail` module. Click on the three dots on the
+upper right of the module and select :guilabel:`Upgrade`.
+
+Desktop App VS Web Application Application Type
+-----------------------------------------------
+
+Should :guilabel:`Desktop App` under the :guilabel:`Application Type` be selected when creating the
+:guilabel:`Credentials` (OAuth client ID), the following error message will display:
+
+.. image:: google_oauth/error-400.png
+   :class: img-fluid
+   :align: center
+   :alt: Error 400 Redirect URI Mismatch.
+
+To correct this error, delete the credentials already created and create new credentials using the
+'Web Application' Application Type.
```

Comment:
Line 228-229: replace `using the 'Web Application' Application Type` with `, selecting :guilabel:'Web Application' for the :guilabel:'Application Type'`